the-collab-lab/tcl-70-smart-shopping-list
# Issue #19 | Recreate Nav bar using MUI

## Description

This code adds a NavBar component that uses MUI's app bar component to create a responsive nav bar.

## Related Issue

closes #42 

## Acceptance Criteria

- [x] Use the Appbar in MUI to display a Navbar

- [x] Implement a hamburger dropdown if Appbar doesnt already do so

## Type of Changes


|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: New feature     |
|  ✓    | 🎆 Enhancement |

## Updates

### Before

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/12a22f5e-2bc5-4dc5-8060-0a9fa8c23f66)


### After

 - Nav bar on medium or larger screens

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/a34b2a3b-7921-44a1-a74f-2821298537ad)

- Nav bar on screens smaller than medium

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/9a339d6c-39b4-4685-b2cc-ecdc58bb36e8)

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/3bd78c4f-3bb4-4874-a7fd-1b03f57d9240)


## Testing Steps / QA Criteria

* From your terminal, pull down this branch with `git pull origin gl-km-mui-nav-bar` and check that branch out with `git checkout gl-km-mui-nav-bar` 
* In the terminal, enter `npm start`
* In the browser, go to localhost:3000 and sign in
* Test the different links in the nav bar to see if they still work
* Using your browsers developer tools, test the responsiveness of the nav bar and click the hamburger to test the links like before.
